Makefile

```diff
@@ -101,6 +101,14 @@ lint: $(GOLANGCI_LINT) ## Lint quickly using `golangci-lint --fast=true`
 lint-full: $(GOLANGCI_LINT) ## Lint thoroughly using `golangci-lint --fase=false`
 	$(GOLANGCI_LINT) run -v --fast=false
 
+.PHONY: lint-focus
+lint-focus: $(GOLANGCI_LINT) ## Lint using specified linter
```

Comment:
Since `hack/verify-all.sh` is already calling one of the other make targets, this target shouldn't be needed.

Reply:
Thanks so much @detiber 
Actually, I've been thinking about doing this for a long time.
After doing this, I feel the code is much cleaner :)